react/lib/components/Widget/AltpaymentWidget.tsx

```
@@ -492,94 +397,79 @@ export const AltpaymentWidget: React.FunctionComponent<AltpaymentProps> = props
+                  <AmountError>Amount is above maximum.</AmountError>
                 )}
               </Fragment>
             ) : (
               <Fragment>
                 {coins.length === 0 && <div>Loading...</div>}
                 {coins.length > 0 && (
                   <Fragment>
-                    <div className={classes.header}>
+                    <Header>
                       Swap coins with
                       <a href="https://sideshift.ai" target="_blank">
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)